index.html

```diff
@@ -202,21 +206,45 @@ <h2>String Searching in Natural Language Content</h2>
-	  sensitivity, or whether the feature supports different aspects of a regular expression language or "wildcards".</p>
-	  
-      <p>A different type of string searching, <strong>which is outside the scope of this document</strong>, is <dfn>Full Text Search</dfn>. When you are using a search engine, you are generally using a form of full text search. Full text search generally breaks natural language text into word segments and may apply complex processing to get at the semantic "root" values of words. For example, if the user searches for "run", you might want to find words like "running", "ran", or "runs" in addition to the actual search term "run". This process, naturally, is sensitive to language, context, and many other aspects of textual variation. </p>
+      <p>There are different types of string searching.</p>
+      
+      <p class="definition"><dfn data-lt="full text search|full-text search|full text searching">Full-Text Search</dfn> refers to searches that process the entire contents of the textual document or set of documents. Full-text queries perform linguistic searches against text data in full-text indexes by operating on words and phrases based on the rules of a particular language such as English or Japanese. Full-text queries can include simple words and phrases or multiple forms of a word or phrase.</p>
+      
+      <p>Frequently this means that a <a>full-text search</a> employs indexes and natural language processing. When you are using a search engine, you are using a form of full text search. Full text search often breaks natural language text into words or phrases (this is called <a>segmentation</a>) and may apply complex processing to get at the semantic "root" values of words (this is called <a>stemming</a>). These processes are sensitive to language, context, and many other aspects of textual variation.</p>
+      
+      <p>One limited form of full-text search&mdash;and the topic of this document&mdash;is sub-string matching. One familiar form of sub-string matching is the "find" feature of your browser. A sub-string match searches the body or <a>corpus</a> of a document with the user's input, seeking a match. Find operations can have different options or implementation details, such as the addition or removal of case sensitivity, or whether the feature supports different aspects of a regular expression language or "wildcards".</p>
+      
+      <p>One way that sub-string matching usually differs from other types of <a>full-text search</a> is that, while it may use algorithms in an attempt to suppress or ignore textual variations, it does not produce matches that contain additional or unspecified character sequences, words, or phrases.</p>
+      
+      <aside class="example" title="Differences between sub-string matching and other types of search">
```

Comment:
What are other types of search?

I guess linked data based semantic search and voice search are out of the scope of this document?

Reply:
Yeah, this is sloppy on my part. My intention is to show the difference between a find type operation and classical FTS, e.g. in Find, the term `space monkey` doesn't match `spaced winged monkeys` while in other forms of full-text search it does.